package.json

```diff
@@ -70,8 +70,7 @@
     "typescript": "5.8.3",
     "typescript-eslint": "^8.31.1",
     "vite": "6.3.4",
-    "vitest": "^3.1.2",
-    "globals": "^16.0.0"
```

Comment:
Accidentally added 2 entries of "globals" and it merged in before I could update it! Hence change here.